sdk/appconfiguration/app-configuration/samples/helloworld.ts

```diff
@@ -0,0 +1,41 @@
+import { getConnectionStringFromEnvironment, deleteKeyAndLabels } from "./sampleHelpers";
+
+// NOTE: replace with import { AppConfigurationClient } from "@azure/app-configuration"
+// in a standalone project
+import { AppConfigurationClient } from "../src"
+
+export async function run() {
+    console.log("Running helloworld sample");
+
+    let connectionString = getConnectionStringFromEnvironment();
+    const client = new AppConfigurationClient(connectionString);
+
+    const greetingKey = "Samples:Greeting";    
+    
+    deleteKeyAndLabels([greetingKey], client);
+
+    // creating a new setting
+    console.log(`Adding in new setting ${greetingKey}`);
+    await client.addConfigurationSetting(greetingKey, { value: "Hello!" });
+
+    const newSetting = await client.getConfigurationSetting(greetingKey);
+    console.log(`${greetingKey} has been set to ${newSetting.value}`);
+
+    // changing the value of a setting
+    await client.setConfigurationSetting(greetingKey, { value: "Goodbye!" });
```

Comment:
Could we show the samples, also using the eTag field? Maybe that should be a separate sample but if that's what we ultimately recommend users do, would be nice to show them how :)

Reply:
I've added a sample that demonstrates the etag (and also tries to explain optimistic concurrency)